rollup.config.js

```diff
@@ -29,6 +30,14 @@ export default {
   ],
   plugins: [
     external(),
+    alias({
```

Comment:
I am actually not sure. I guess it would make sense that it is not needed, as long as none of the dependencies in the project use React, which should be the case.